howso/engine/trainee.py

```diff
@@ -3200,6 +3200,74 @@ def get_marginal_stats(
         else:
             raise AssertionError("Client must have the 'get_marginal_stats' method.")
 
+    def get_value_masses(
+        self,
+        features: Collection[str],
```

Comment:
Would be nice if this could be overloaded so if you only specify a single feature you just get those values back.